website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -19,6 +19,8 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 [^*] The official release date for this new format of release notes is May 15th, 2024. Historical release notes for prior dates may not reflect all available features released earlier this year or their tenancy availability.
 
 ## July 2024
+- **Behavior change:** dbt Cloud IDE attaches a `--limit 100` to queries by default to avoid slow and expensive queries from running in the development environment. Recently, dbt Core has changed how the `limit` is applied to ensure that `order by` clauses are consistently respected. Because of this, queries that already contain a limit clause might now cause errors in the IDE previews. An option to prevent the limit from being applied will become available to the dbt Cloud IDE later this year. Until this is released, dbt Labs strongly recommends removing the (duplicate) limit clause from your queries when previewing them.
```

Comment:
```suggestion
- **Behavior change:** dbt Cloud IDE attaches a `--limit 100` to preview queries by default to avoid slow and expensive queries from running in the development environment. Recently, dbt Core has changed how the `limit` is applied to ensure that `order by` clauses are consistently respected. Because of this, queries that already contain a limit clause might now cause errors in the IDE previews. An option to prevent the limit from being applied will become available to the dbt Cloud IDE later. Until this is released, dbt Labs recommends removing the (duplicate) limit clause from your queries when previewing them.
```

"later this year" sounds a long time away, but this should be pretty soon 🤞 
I wouldn't say we have to _strongly_ recommend it - if you don't it just won't work. Feels like we only have to be that wemphatic when you might be about to do something possible but ill-advised

Reply:
sgtm! thank you!